src/WinRT.Runtime/Projections/Nullable.cs

```diff
@@ -1869,8 +1968,8 @@ public ComWrappers.ComInterfaceEntry[] GetExposedInterfaces()
 
         public static ComWrappers.ComInterfaceEntry[] GetExposedInterfaces(ComWrappers.ComInterfaceEntry delegateInterface)
         {
-            return new ComWrappers.ComInterfaceEntry[]
-            {
+            return
+            [

```

Comment:
This syntax is C# 12, should we not avoid using it until we drop downlevel support, because these files could also be used in the embedded scenario, and people might not be using C# 12 there?

Reply:
That is true, will address.